### DISCUSSION TITLE
Practica css

### DISCUSSION THREAD

**Discussion by DaveOval** [Launch-X-Latam/MisionFrontEnd#467] [Team Posts]
Que onda camaradas, queria preguntar si alguien ya empezo la practica de css de vaccination para saber si conocen la fuente que utilizan y si saben en donde puedo sacar esas fotos que usan ya que ando intentando buscar unas iguales y nomas nada saludos.

**Comment by AlexLopez24** [comment]
Hay una extension de google chrome que se llama whatFont que te puede servir para la fuente de letra, y las imagenes las puedes conseguir en https://www.pexels.com/es-es/, lo que  yo hice fue poner algo tipo "enfermero fondo azul" y me mostraba imagenes similares a la del template que nos dejaron.

**Comment by leandrho** [comment]
Estoy por empezar con la practica CSS y me preguntaba los mismo. Gracias por la Data. Saludos

**Comment by floressdi** [comment]
¿Hola en donde se encuentra la practica de los pasteles?

**Comment by Rarriaza-sv** [comment]
https://github.com/orgs/Launch-X-Latam/teams/mision-frontend/discussions/40
